src/containers/IntegrationLander.tsx

```diff
@@ -48,6 +49,8 @@ export default function render() {
           </div>
         </div>
 
+        {tags && tags.some(t=>t==="deprecated") && <div className="alert alert-warning">This integration has been <b>deprecated</b>. It will continue working for exitsing customers, but is not recommended for new customers.</div>}
```

Comment:
```suggestion
        {tags && tags.some(t=>t==="deprecated") && <div className="alert alert-warning">This integration has been <b>deprecated</b>. It will continue working for existing customers, but is not available to be installed by new customers.</div>}
```